tests/tests.py

```diff
@@ -1939,6 +1956,63 @@ def test_session_server_offline(self):
         self.assertTrue(False, "Instantiation of invalid JIRA instance succeeded.")
 
 
+class AsyncTests(unittest.TestCase):
+
+    def setUp(self):
+        self.jira = JIRA('https://jira.atlassian.com', logging=False,
+                         async=True, validate=False, get_server_info=False)
+
+    def test_fetch_pages(self):
```

Comment:
This test will not pass without 086afbbc96c1f981b997cf9e1351bda8ba81ed6e and 05e8b09456eaa9b3a2742033cc145e841bf1822e